java/ql/src/semmle/code/java/dataflow/internal/ContainerFlow.qll

```diff
@@ -222,23 +240,38 @@ private predicate taintPreservingArgumentToQualifier(Method method, int arg) {
   method.(CollectionMethod).hasName("offer") and arg = 0
   or
   // java.util.Deque
+  // covered by Stack: push(E)
   method.(CollectionMethod).hasName(["addFirst", "addLast", "offerFirst", "offerLast"]) and arg = 0
   or
   // java.util.concurrent.BlockingQueue
+  // covered by Queue: offer(E, long, TimeUnit)
   method.(CollectionMethod).hasName("put") and arg = 0
   or
   // java.util.concurrent.TransferQueue
   method.(CollectionMethod).hasName(["transfer", "tryTransfer"]) and arg = 0
   or
   // java.util.concurrent.BlockingDeque
+  // covered by Deque: offerFirst(E, long, TimeUnit), offerLast(E, long, TimeUnit)
   method.(CollectionMethod).hasName(["putFirst", "putLast"]) and arg = 0
+  or
+  //java.util.Dictionary
```

Comment:
Would probably be good to add a space between `//` and `java...` to be consistent:
```
// java.util.Dictionary
```